force-app/main/default/classes/PS_IntegrationServiceConfig_TEST.cls

```diff
@@ -96,18 +98,30 @@ public with sharing class PS_IntegrationServiceConfig_TEST {
 
     @IsTest
     private static void shouldHaveIntegrationDisabledWhenIsActiveServiceRecordIsFalse() {
-        setUpConfiguration();
-        insert new Payment_Services_Configuration__c(
-            Service__c = PS_IntegrationServiceConfig.PAYMENTS_SERVICE_NAME,
-            Key__c = PS_IntegrationServiceConfig.IS_ACTIVE,
-            Value__c = 'false'
-        );
-
+        PaymentServicesConfigurationSelectorMock configurationSelectorMock =
+            new PaymentServicesConfigurationSelectorMock()
+                .withClientId(true)
+                .withProductId(false)
+                .withActiveFlagAsFalse(true);
+        PS_IntegrationServiceConfig.paymentServicesConfigurationSelector = stubFor(configurationSelectorMock);
         PS_IntegrationServiceConfig ps = new PS_IntegrationServiceConfig();
         System.assertEquals(false, ps.isIntegrationEnabled(),
             'The org should have Integration Disabled when isActive record is false');
     }
 
+    @IsTest
+    private static void shouldHaveIntegrationDisabledWhenIsActiveServiceRecordIsTrue() {
```

Comment:
Should this one be "Enabled" instead of disabled (in the method name)?

Reply:
Pushed up an update